buildSrc/src/main/java/Deps.kt

```diff
@@ -177,4 +177,10 @@ object Deps {
       const val Tape = "com.squareup:tape:$version"
     }
   }
+
+  object Spencerwi {
+    private const val version = "2.5.0"
+
+    const val Either = "com.spencerwi:Either.java:$version"
```

Comment:
For the next commit that will implement the core of "fetching with time-budget" logic, I will need an `Either` implementation (akin to what exists in functional languages like Haskell).

Reply:
Please note that this library is using Java 8 language level. I'm ok with with, but this may be mentioned in the CHANGELOG.

Also, I saw that this lib is only implementing an Either. What about importing a FP lib instead, so if we want to grab some other FP concept, they will be interconnected ? Like [ArrowKt](https://arrow-kt.io/docs/0.10/apidocs/arrow-core-data/arrow.core/-either/).